### ISSUE TITLE
Problem using Plugin Search

### ISSUE THREAD

**Issue by yungHa** [VundleVim/Vundle.vim#906]

**Comment by davidlowryduda** [comment]
I confirm this behavior on my machine. Although I've used vundle for a very long time, I've don't use PluginSearch and never noticed.

The root of this problem is that PluginSearch queries http://vim-scripts.org, and this site no longer exists. This is because most vim plugin development and distribution occurs via github/gitlab now.

I recommend searching for plugins directly, or searching for plugins on github. This functionality should be considered dead now.

**Comment by yungHa** [comment]
> I confirm this behavior on my machine. Although I've used vundle for a very long time, I've don't use PluginSearch and never noticed.
> 
> The root of this problem is that PluginSearch queries http://vim-scripts.org, and this site no longer exists. This is because most vim plugin development and distribution occurs via github/gitlab now.
> 
> I recommend searching for plugins directly, or searching for plugins on github. This functionality should be considered dead now.

Thanks for the reply, which means all I have to do now is to find the plugin. Put "Plugin *plugin_name*" into my vimrc and do "PluginInstall"?

I haven't used Vundle before , any help would be appreciated

**Comment by davidlowryduda** [comment]
>Thanks for the reply, which means all I have to do now is to find the plugin. Put "Plugin plugin_name" into my vimrc and do "PluginInstall"?

Yes, that's right. For instance, if you looked on google for a vim git plugin and you found that people use Tim Pope's fugitive, at https://github.com/tpope/vim-fugitive, then you could add this plugin to your setup by adding the line

    Plugin 'tpope/vim-fugitive'

between the lines containing `call vundle#begin` and `call vundle#end` (in practice, immediately after the line containing `Plugin VundleVim/Vundle.vim`). This general format of referring to github locations for vim plugins has become very common. There are a few plugins that might be desirable but which are not on github --- if you come across one of these and need help, raise an issue here and I'm sure some help will come.

>I haven't used Vundle before , any help would be appreciated

Good luck! So that you know, vundle is a very simple plugin manager (which is why I like it). There are other plugin managers that are more featured (if that's something you think you need). vim-plug is both more featureful and has a more active community of developers right now than vundle. (But I like and use vundle --- its simplicity is all I'm looking for).

**Comment by habsfanongit** [comment]
You can search plugins on https://vimawesome.com and install them manually with PluginInstall
